cppclient/.bazelrc

```diff
@@ -0,0 +1 @@
+build - -copt - w
```

Comment:
What is it for ?
And shouldn't it be `build --copt -w` ?

Reply:
it's not used because it's not imported, good catch, i will remove